setup.py

```diff
@@ -62,7 +62,43 @@ def get_git_sha():
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
-    entry_points={"console_scripts": ["superset=superset.cli:superset"]},
+    entry_points={
+        "console_scripts": ["superset=superset.cli:superset",],
+        "superset.db_engine_specs": [
+            "athena=superset.db_engine_specs.athena:AthenaEngineSpec",
+            "bigquery=superset.db_engine_specs.bigquery:BigQueryEngineSpec",
+            "clickhouse=superset.db_engine_specs.clickhouse:ClickHouseEngineSpec",
+            "cockroachdb=superset.db_engine_specs.cockroachdb:CockroachDbEngineSpec",
+            "crate=superset.db_engine_specs.crate:CrateEngineSpec",
+            "databricks=superset.db_engine_specs.databricks:DatabricksHiveEngineSpec",
+            "db2=superset.db_engine_specs.db2:Db2EngineSpec",
+            "dremio=superset.db_engine_specs.dremio:DremioBaseEngineSpec",
+            "drill=superset.db_engine_specs.drill:DrillEngineSpec",
+            "druid=superset.db_engine_specs.druid:DruidEngineSpec",
+            "elasticsearch=superset.db_engine_specs.elasticsearch:ElasticSearchEngineSpec",
+            "odelasticsearch=superset.db_engine_specs.elasticsearch:OpenDistroEngineSpec",
+            "exasol=superset.db_engine_specs.exasol:ExasolEngineSpec",
+            "firebird=superset.db_engine_specs.firebird:FirebirdEngineSpec",
+            "gsheets=superset.db_engine_specs.gsheets:GSheetsEngineSpec",
+            "hana=superset.db_engine_specs.hana:HanaEngineSpec",
+            "hive=superset.db_engine_specs.hive:HiveEngineSpec",
+            "impala=superset.db_engine_specs.impala:ImpalaEngineSpec",
+            "kylin=superset.db_engine_specs.kylin:KylinEngineSpec",
+            "mssql=superset.db_engine_specs.mssql:MssqlEngineSpec",
+            "mysql=superset.db_engine_specs.mysql:MySQLEngineSpec",
+            "oracle=superset.db_engine_specs.oracle:OracleEngineSpec",
+            "pinot=superset.db_engine_specs.pinot:PinotEngineSpec",
+            "postgres=superset.db_engine_specs.postgres:PostgresEngineSpec",
+            "presto=superset.db_engine_specs.presto:PrestoEngineSpec",
+            "redshift=superset.db_engine_specs.redshift:RedshiftEngineSpec",
+            "snowflake=superset.db_engine_specs.snowflake:SnowflakeEngineSpec",
+            "solr=superset.db_engine_specs.solr:SolrEngineSpec",
+            "sqlite=superset.db_engine_specs.sqlite:SqliteEngineSpec",
+            "teradata=superset.db_engine_specs.teradata:TeradataEngineSpec",
+            "trino=superset.db_engine_specs.trino:TrinoEngineSpec",
+            "vertica=superset.db_engine_specs.vertica:VerticaEngineSpec",
+        ],
+    },
```

Comment:
Would it be possible to automate this part by iterating over modules in `superset/db_engine_specs/*.py` and checking that they are `issubclass` but not `isinstance` of `BaseEngineSpec`?

Reply:
Will do!